step_definitions/teams/createTeam_steps.js

```diff
@@ -5,8 +5,8 @@ const TMSTeamListPage = require('../../page-objects/pages/teamsPages/TMSTeamList
 const TMSTeamMembersPage = require('../../page-objects/pages/teamsPages/TMSTeamMembersPage.js');
 
 //WHEN
-When(/^.*creates a new team with name '([^']*)' and description '([^']*)' and color orange$/, function (teamName, description) {
-	return TMSAddEditTeamPage.createTeamAndGoToInternalMembersManagement(teamName, description);
+When(/^.*creates a new team with name '([^']*)' and description '([^']*)' and color '([^']*)'$/, function (teamName, description, colour) {
+	return TMSAddEditTeamPage.createTeamAndGoToInternalMembersManagement(teamName, description, colour);
```

Comment:
```suggestion
	await TMSAddEditTeamPage.createTeamAndGoToInternalMembersManagement(teamName, description, colour);
```
The same in other similar places